concordium-consensus/tests/consensus/ConcordiumTests/KonsensusV1/LMDB.hs

```diff
@@ -86,7 +87,7 @@ dummyBlockSig = Block.sign dummyKP "someMessage"
 -- | A helper function for creating a 'BakedBlock' given a round. Used by 'dummyBlock' to create blocks.
 --  The block is well-formed and contains the supplied transactions and is for the specified round.
 --  Beyond that, there should be no expectation on the data in the block.
-dummyBakedBlock :: Round -> Vector.Vector BlockItem -> BakedBlock
+dummyBakedBlock :: Round -> Vector.Vector BlockItem -> BakedBlock 'P6
```

Comment:
Is there a reason why we don't have one such for P7?

Reply:
This is now generalized (as part of #1111)